wcvendors/wcvendors
# WIP: Add test cases for functions file and backfill email

### All Submissions:






### Changes proposed in this Pull Request:



 Added:

- tests/phpunit/testcases/includes/wcv-admin-functions-test.php
- tests/phpunit/testcases/includes/wcv-core-functions-test.php
- tests/phpunit/testcases/includes/wcv-vendor-functions-test.php

### How to test the changes in this Pull Request:

1. Clone the next branch
2. Run 'npm start' command
3. Run 'vendor/bin/phpunit' command
4. See no test case failed
### Other information:
